### ISSUE TITLE
Recent use cases

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act#301]
## Use case 1:

Implementors of rules want to know the relationship to WCAG success criteria, for rules that do not directly test that success criteria (aka best practices). For example, valid HTML is a technique for WCAG 2 SC 4.1.1, but having in valid HTML does not mean that success criterion wasn't met. How can this relationship be conveyed in a consistent way?

## Use case 2:

Content authors want to know if failing the rule means they've "failed" WCAG (non-conformance). This can be difficult, as requires proper understanding of terms such as sufficient, advisory, guideline, normative, etc. An additional complication is that some sites have accessibility requirements that go beyond WCAG success criteria, while others use a subset of WCAG success criteria.

## Use case 3:

Writers of an accessibility requirements document, such as someone looking to have a website developed by a third party, or someone defining a standard for all websites owned by an organisation, want to use rules as part of their requirement. Since the author of the accessibility requirements document may not be the author of the rule, they can not add their requirements document to the rule as "accessibility requirements".

**Comment by nitedog** [comment]
- I don't fully understand "use case 3" - I think this is about extensibility of the metadata?
- Another use case is that some people want to be able to (easily) identify if a rule fully addresses a requirement, or if further testing is needed if the rule passes - Maybe this relates to use cases 1 & 2

**Comment by nitedog** [comment]
> @nitedog Can you please rephrase this without the word "metadata". That is not something we've defined in ACT Rules, so it's hard to know what you're asking for without me having to make a lot of assumptions.

I don't fully understand "use case 3".

> This has been part of the rules format for a long time now. Can you describe to me for who the current solution is insufficient, and why? Otherwise I don't think there's anything to be done there.

How does one express that a rule is only a partial check for the requirement?

**Comment by WilcoFiers** [comment]
I've updated the example. Hope that clarifies it a bit. As for "how", the current draft doesn't specify that (nor does it specify anywhere else "how" to do something, just "what" to do). It says this:

> Because ACT Rules often have a smaller scope than the accessibility requirement they test, passing a rule does not necessarily mean that the accessibility requirement has passed.

**Comment by annethyme** [comment]
> ACT Rules MUST indicate when they can not be used to determine that the accessibility requirement passed.

I guess one of the problems is that no proposed ACT Rule has actually ever indicated this.
As I have experienced it, it has always been an implicit assumption that no rule tests a whole requirement. 

And we have actually in Auto-WCAG been telling some rule authors that they were not allowed to write this out explicitly in rules.

**Comment by skotkjerra** [comment]
The ACT rules format introduction section specifies that the purpose of the format is to enable a consistent interpretation of how to test for accessibility requirements. We specify what an ACT rule should contain, and rather than how the information is collected, we specify the structure of the information - e.g. where/how it should be noted/indicated/conveyed.

We say that ACT Rules MUST indicate when they can not be used to determine that the accessibility requirement passed. My opinion is that if we do so, we also need to state clearly where in an ACT rule this should be indicated - to ensure consistent interpretation of the rules themselves.

The goal of the ACT TF is to define a format for ACT-rules, and how things have been practiced in the Auto-WCAG CG does not necessarily dictate how the ACT format should be changed, but of course we should collect learnings from auto-WCAG to make the format as easy to use as possible, and the fact that there has been a discussion on just this subject seems to indicate that there is a challenge.

**Comment by skotkjerra** [comment]
My take on this is +1 to the suggestions outlined in https://github.com/w3c/wcag-act/issues/290
This suggestion addresses the need from the use cases above.

**Comment by WilcoFiers** [comment]
We did a rewrite of the accessibility requirements section in #313. I believe this addresses all issues raised here.
